cloudflare/resource_cloudflare_waf_package.go

```diff
@@ -0,0 +1,255 @@
+package cloudflare
+
+import (
+	"fmt"
+	"strings"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceCloudflareWAFPackage() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudflareWAFPackageCreate,
+		Read:   resourceCloudflareWAFPackageRead,
+		Update: resourceCloudflareWAFPackageUpdate,
+		Delete: resourceCloudflareWAFPackageDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceCloudflareWAFPackageImport,
+		},
+
+		SchemaVersion: 0,
+		Schema: map[string]*schema.Schema{
+			"package_name": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+
+			"package_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+
+			"zone": {
```

Comment:
We aren't supporting `zone` for old resources as of 2.0 so let's just drop it here entirely and only support zone ID going forward.